news/2019-11-19-osu-idol-2019-finals-community-voting.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "osu!idol 2019 Finals: Community Voting"
+date: 2019-11-19 12:00:00 +0000
+---
+
+It's been another amazing year for the osu!idol competition! There are three finalists we have for you to enjoy: [aceredshirt13](https://osu.ppy.sh/users/13736238), [moof](https://osu.ppy.sh/users/8457202), and [iamALieee](https://osu.ppy.sh/users/10093980). Go give their entries a listen and vote for your favourite!
+
+![](/wiki/shared/news/2019-11-19-osu-idol-2019-finals-community-voting/banner.jpg)
+
+osu!idol is a singing contest driven by the community where contestants can participate solo or as a duet and sing their way through multiple stages that consist of different themes and genres to see who will reach the top! Now it is up to you to decide who will be the winner!
```

Comment:
```suggestion
osu!idol is a contest driven by the community where contestants sing their way through multiple stages to see who will reach the top. Now it's up to you to decide who will be the winner!
```
first sentence is a super run-on, could cut out these details cuz they aren't relevant to finals voting